modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc

```diff
@@ -230,12 +230,14 @@ image::ms-dynamics/ms-service-bus-studio-1.png[General]
 
 == Use Case: Get Queue list
 
+This use case retrieves the list of existing queues in a specific namespace. It returns the list in JSON format.
+
 image::ms-dynamics/ms-service-bus-flow.png[General]
 
-. Create a new Mule Application on your Studio and select an HTTP Listener as a Source in the new flow.
-. Add a new HTTP Listener Configuration global element:
+. Create a new Mule Application in Studio and select HTTP Listener as a source in the new flow.
+. Add a new HTTP Listener configuration global element:
 +
-. Specify Host and port parameters with the following values:
+. Specify the Host and Port parameters with the following values:
```

Comment:
ditto: code parameters should be ` `; UI labels should be * *.
I will stop marking.